src/components/ZetkinPersonLink.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from '@material-ui/core';
```

Comment:
This is nice - can it be copied and pasted elsewhere in the app? If so, could add an issue to do that later, or include it as part of this PR?

Reply:
I don't know if we have simple text links like this elsewhere, but if we do it could absolutely be reused.